index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Capstone1: homepage</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="first-section">
+    <header>
+      <nav>
+        <img src="images/mobile_menu.png" class="menu" id= "menu-mobile" alt="menu">
+        <img src="images/logo3.png" alt="logo" class="logo">
+        <div class="desktop-menu">
+          <ul class="links">
+            <li><a href="About.html">About</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#leaders">Leaders</a></li>
+            <li><a href="#partners">Partners</a></li>
+          </ul>
+          <button class="signup" type="button">
+            Sign Up
+          </button>
+        </div>
+         <!-- Mobile-menu -->
+        <div id="mobile-menu">
+          <a id="menu-exit">&times;</a>
+          <ul id="menu-items">
+            <li class="menu-link"><a href="About.html">About</a>
+            </li>
+            <li class="menu-link" ><a href="#program">Program</a>
+            </li>
+            <li class="menu-link"><a href="#leaders">Leaders</a>
+            </li>
+            <li class="menu-link"><a href="#partners">Partners</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
```

Comment:
- Kindly add the black header for the desktop version to match the Behance design in the desktop version.
- Kindly make the `ul` tag that contains `li` tags centered vertically as the space on the bottom is bigger than the space on the top.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157787557-d4c7599e-2712-469c-8a4a-e3e40ebafd7a.png) | ![image](https://user-images.githubusercontent.com/24703171/157787611-c0b91a57-4c8c-4dce-a7ae-578acf6d893c.png) |

Reply:
Addressed!